pontoon/settings/base.py

```diff
@@ -61,6 +61,7 @@ def path(*args):
 
 SESSION_COOKIE_HTTPONLY = os.environ.get('SESSION_COOKIE_HTTPONLY', 'True') != 'False'
 SESSION_COOKIE_SECURE = os.environ.get('SESSION_COOKIE_SECURE', 'True') != 'False'
+CSRF_COOKIE_SECURE = os.environ.get('CSRF_COOKIE_SECURE', 'True') != 'False'
```

Comment:
Looking at the [commonware source code](https://github.com/mozilla/commonware/blob/master/commonware/response/cookies/monkeypatch.py), it seems like we also need to add the `CSRF_COOKIE_HTTPONLY` setting?